packages/@aws-cdk/aws-cloudfront/test/integ.distribution-policies.ts

```diff
@@ -0,0 +1,19 @@
+import * as cdk from '@aws-cdk/core';
+import * as cloudfront from '../lib';
+import { TestOrigin } from './test-origin';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integ-distribution-policies');
```

Comment:
This is a weird name for this file (and this Stack)... was this supposed to be something else originally?

Reply:
My thought is this integ test will show how both policies (Origin Request and Cache) can be created and attached to a distribution. The other integ tests are all named `integ.distribution-something.ts`, so here the _something_ is testing the policies. :shrug:. Open to alternative naming suggestions.

Reply:
I was just surprised to not see the word 'cache' anywhere there. But I don't care that much 🙂.